src/nrnoc/extcelln.cpp

```diff
@@ -34,10 +34,10 @@ static const char* mechanism[] = {
     "vext[N]",
     0,
 };
-static HocParmLimits limits[] = {{"xraxial", 1e-9, 1e15},
-                                 {"xg", 0., 1e15},
-                                 {"xc", 0., 1e15},
-                                 {0, 0., 0.}};
+static HocParmLimits limits[] = {{"xraxial", {1e-9, 1e15}},
+                                 {"xg", {0., 1e15}},
+                                 {"xc", {0., 1e15}},
+                                 {nullptr, {0., 0.}}};
```

Comment:
shouldn't we have 
```suggestion
                                 {"", {0., 0.}}};
```

Reply:
I just preserved how it was before. Looks like it's important that it's a null pointer https://github.com/neuronsimulator/nrn/blob/3ddf92ca540000e6435e1cf19126e77011a7f795/src/nrnoc/init.cpp#L876 instead of an empty string.